### DISCUSSION TITLE
Add possibility to select domain when creating a random alias

### DISCUSSION THREAD

**Discussion by lucasmchoi** [simple-login/app#1549] [Feature Requests]
Currently one must change the default domain used for creating a random alias in the settings. It would be nice to be able to choose directly when creating a random alias. Maybe as a dropdown in the same dialogue where you can choose between all domains and custom domains.

**Comment by Ambiguity4142** [comment]
Having several custom domains, I agree this would be a very nice feature. A drop down list to choose the domain to create the alias with would be perfect.

**Comment by alexamiryan** [comment]
Please make it possible to generate a random alias on non default domain. Right now Random alias generation button both on web interface and browser extension generates a random alias on a pre-selected default domain. I have several custom domain attached to my account and there is no way I can generate random aliases on other domains.

**Reply by nguyenkims** [reply]
You can change the default domain for random alias using this setting: 
![CleanShot 2023-02-15 at 11 43 37@2x](https://user-images.githubusercontent.com/3167780/219006003-2097d4f1-59f7-4e1a-8a83-56447bfc8225.png)

**Reply by alexamiryan** [reply]
> You can change the default domain for random alias using this setting: ![CleanShot 2023-02-15 at 11 43 37@2x](https://user-images.githubusercontent.com/3167780/219006003-2097d4f1-59f7-4e1a-8a83-56447bfc8225.png)

I know about this option, and the whole point is that you have to change the default domain when you want to generate a random alias on other domain. 
Feature request is to have a dropdown menu with available domains on random domain generation screen, both on web interface and Firefox/Chrome extension.
